core/common/src/main/java/alluxio/conf/InstancedConfiguration.java

```diff
@@ -301,6 +301,13 @@ public int getInt(PropertyKey key)
     return (int) get(key);
   }
 
+  @Override
+  public long getLong(PropertyKey key)
+  {
+    checkArgument(key.getType() == PropertyKey.PropertyType.INTEGER);
```

Comment:
It should be `PropertyKey.PropertyType.INTEGER` or `PropertyKey.PropertyType.LONG` ?